include/libopencm3/stm32/l4/memorymap.h

```diff
@@ -106,6 +106,7 @@
 #define GPIO_PORT_H_BASE		(PERIPH_BASE_AHB2 + 0x1c00)
 /* Still AHB2, good job ST */
 #define OTG_FS_BASE			(0x50000000U + 0x00000)
+#define USB_OTG_FS_BASE     OTG_FS_BASE
```

Comment:
as OTG was never enabled on l4, just drop the original defintion, and use the "standard" definitoin please.

Reply:
Ok, done